### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/features.md`

### PR DESCRIPTION
**Pull request by kty4119**
* Add Korean translation for docs/ko/docs/features.md
* Related [FEATURE] Korean translations [FEATURE] Korean translations #2017

### CODE REVIEW

File: docs/ko/docs/features.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+## FastAPI 기능
+
+**FastAPI** 에는 다음과 같은 기능이 있습니다:
+
+### 개방형 표준 준수
+
+- API 작성을 위한 [**OpenAPI**](https://github.com/OAI/OpenAPI-Specification) 에는, 경로 작업 선언, 매개 변수, 본문 요청, 보안 등이 포함됩니다.
+- [**JSON Schema**](https://json-schema.org/) 를 사용한 데이터 모델의 문서를 자동 생성합니다(OpenAPI는 JSON 스키마를 기반으로 함).
+- 면밀히 조사의 결과, 상층에 개설하는 것이 아니라, 이 기준에 그거해 설계되었습니다.
```
Comment (DevDae) [inline docs/ko/docs/features.md:11]:
```suggestion
- 면밀히 조사의 결과, 상층에 개설하는 것이 아니라, 이 기준에 의거해 설계되었습니다.
```
Maybe you make mistake, what's mean of the word, "그거해"? 😄

File: docs/ko/docs/features.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+그러나 기본적으로, 모든 기능이 **"잘 작동합니다"**.
+
+### 검증
+
+- 대부분 (또는 모두?)에 대한 파이썬 **데이터 유형** 유효성검사는, 다음을 포함합니다:
+  - JSON 객체 (`dict`).
+  - 항목 유형을 정의하는 JSON 배열 (`list`).
+  - 최소 길이와 최대 길이가 있는 문자열 (`str`) 필드.
+  - 최소값과 최대값이 있는 숫자 (`int`, `float`), 등.
+- 보다 이국적인 유형에 대한 검증, 예를 들면:
```
Comment (DevDae) [inline docs/ko/docs/features.md:109]:
```suggestion
- 보다 익숙하지 않은 유형에 대한 검증, 예를 들면:
```
I think "익숙하지 않은" can represent the meaning of the "exotic" than "이국적인".

**Review by DevDae — CHANGES_REQUESTED** [review]
First of all, i'm really gratitude for your works!
Feel free to my opinions and please give your idea 😁

**Comment by joonas-yoon** [comment]
Oh my. I didn't know about this PR.

I requested new PR and it has no conflict to merge into master branch - https://github.com/tiangolo/fastapi/pull/5237

File: docs/ko/docs/features.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+- 면밀히 조사의 결과, 상층에 개설하는 것이 아니라, 이 기준에 의거해 설계되었습니다.
+- 이것은 또한 많은 언어에서 자동 **클라이언트 코드 생성**을 허용합니다.
+
+### 자동 문서 생성
+
+대화형 API 문서와 탐색적인 웹 사용자 인터페이스를 제공합니다. 프레임워크가 OpenAPI를 기반으로 하기 때문에, 몇 가지 옵션이 있으며, 기본적으로 두 가지가 포함되어 있습니다.
+
+- [**Swagger UI**](https://github.com/swagger-api/swagger-ui), 에서 대화형 탐색을 통해, 브라우저에서 직접 API를 호출하고 테스트할 수 있습니다.
+
+[![Swagger UI interaction](https://camo.githubusercontent.com/b8ead36860df4f2befd2203bfa1da701c57c09ceb16e4ac6b9881d698e9950ce/68747470733a2f2f666173746170692e7469616e676f6c6f2e636f6d2f696d672f696e6465782f696e6465782d30332d737761676765722d30322e706e67)](https://camo.githubusercontent.com/b8ead36860df4f2befd2203bfa1da701c57c09ceb16e4ac6b9881d698e9950ce/68747470733a2f2f666173746170692e7469616e676f6c6f2e636f6d2f696d672f696e6465782f696e6465782d30332d737761676765722d30322e706e67)
```
Comment (tiangolo) [inline docs/ko/docs/features.md:20]:
```suggestion
![Swagger UI interaction](https://fastapi.tiangolo.com/img/index/index-03-swagger-02.png)
```

For these files you can keep the original ones instead.

**Review by tiangolo — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit ce3ce9e8696dae4f071465d7704c3da0fc31753a at: https://649a1544ce5a7609ebde0faf--fastapi.netlify.app

**Pull request closed without merging**